test/test_data/javascript_codegen_example.js

```diff
@@ -0,0 +1,52 @@
+// -----------------------------------------------------------------------------
```

Comment:
This should probably go in `symforce_function_codegen_test_data`?  And use the pattern in the other codegen tests so it automatically generates on both backends